### PR TITLE
Feature/dg light

### PR DESCRIPTION
**Pull request by eskilsen**
## Description

Make possible to import a specific component from NPM.
Less files are now not depended on DG as a whole. 

## How Has This Been Tested?

Updated snapshots and ran all tests. Removed unused and old test code. 
Created several beta packages and tested the components as external users. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/less/components/payex/links.less [hunk 1]
```diff
@@ -10,7 +10,7 @@ a {
     }
 
     &.icon-link {
-        color: @link-color;
+        color: @brand-primary;
 
         &:hover {
             color: @brand-primary;
```
Comment (hmosvoll) [inline src/less/components/payex/links.less:16]: Weird with @brand-primary on hover and icon-link default

File: src/less/components/payex/logo.less [hunk 2]
```diff
@@ -1,24 +1,3 @@
...
-        &.topbar-logo {
-            img {
-                &.logotype-md {
-                    margin-top: 15%;
-                }
-            }
-        }
-    }
-}
-
```
Comment (hmosvoll) [inline src/less/components/payex/logo.less:21]: 👍

File: src/less/components/payex/radiobutton.less [hunk 3]
```diff
@@ -1,3 +1,6 @@
+@import "./form.less";
+@import "../radio.less";
+
```
Comment (hmosvoll) [inline src/less/components/payex/radiobutton.less:3]: Not related pr, but could this easily be renamed to radio? To be consistent

File: src/less/components/rangeslider.less [hunk 4]
```diff
@@ -191,5 +202,27 @@
             display: none;
         }
     }
+
+    //cursor: not-allowed;
```
Comment (hmosvoll) [inline src/less/components/rangeslider.less:206]: rm?

File: src/less/components/skip-link.less [hunk 5]
```diff
@@ -6,6 +6,7 @@
     z-index: -999;
     padding: 0.75rem;
     background-color: @white;
+    font-family: Arial, sans-serif;
```
Comment (hmosvoll) [inline src/less/components/skip-link.less:9]: Could this use a variable?

File: src/less/core/all.less [hunk 6]
```diff
@@ -0,0 +1,6 @@
+@import "grid";
+@import "iconography";
+@import "logotype";
+@import "page-layout";
+@import "typography";
+@import "normalize";
```
Comment (hmosvoll) [inline src/less/core/all.less:6]: 👍

File: src/less/core/iconography.less [hunk 7]
```diff
@@ -1,5 +1,3 @@
-@import "flag-list";
```
Comment (hmosvoll) [inline src/less/core/iconography.less:1]: should this be kept in it's own file to keep the code more readable?

File: src/less/core/normalize.less [hunk 8]
```diff
@@ -1,27 +1,19 @@
 /*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (hmosvoll) [inline src/less/core/normalize.less:1]: Maybe add some thing about that this is a modified version of normalize now

**Review by hmosvoll — COMMENTED** [review]
108 / 144 files reviewed so far

_review done in slack huddle_

File: src/less/components/form.less [hunk 9]
```diff
@@ -1,3 +1,39 @@
+@import "../global.less";
+@import "../core/normalize.less";
```
Comment (hmosvoll) [inline src/less/components/form.less:2]: Why do we import normalize into some component, and not others?

File: src/less/components/input-group.less [hunk 10]
```diff
@@ -64,13 +70,13 @@
         font-size: @base-font-size;
         font-weight: @font-weight-normal;
         line-height: @input-line-height;
-        color: @medium-brown;
+        color: #72605e;
```
Comment (hmosvoll) [inline src/less/components/input-group.less:73]: Should this be a variable for theming?

File: src/less/components/input-group.less [hunk 11]
```diff
@@ -64,13 +70,13 @@
...
         background-color: @white;
         text-align: center;
         white-space: nowrap;
         border: solid @input-border-color;
         border-width: @input-border-width;
-        border-bottom-color: @medium-brown;
+        border-bottom-color: #72605e;
```
Comment (hmosvoll) [inline src/less/components/input-group.less:79]: Should this be a variable for theming?

File: src/less/components/pagination.less [hunk 12]
```diff
@@ -12,7 +14,7 @@
         @media screen and (max-width: (@screen-sm-min - 1)) {
             display: block;
             align-self: center;
-            font-weight: @font-weight-bold;
+            font-weight: 700;
```
Comment (hmosvoll) [inline src/less/components/pagination.less:17]: Should we keep this variable?

File: src/less/payex.less [hunk 13]
```diff
@@ -13,16 +13,14 @@
 @import "components/payex/links.less";
 @import "components/payex/loader.less";
 @import "components/payex/logo.less";
-@import "components/payex/material-icons.less";
 @import "components/payex/progress-step.less";
-@import "components/payex/radiobutton.less";
+@import "components/payex/radio.less";
```
Comment (hmosvoll) [inline src/less/payex.less:17]: 👍

File: src/scripts/main/dialog/index.js [hunk 14]
```diff
@@ -17,7 +17,7 @@ class Dialog {
         this.id = el.id;
         this.closeIcon = el.querySelector(SELECTORS.CLOSEICON);
         this.header = el.querySelector(".dialog-header");
-        this.isOpen = el.classList.contains("d-flex");
+        this.isOpen = el.style.display === "flex"; // this must be rewritten
```
Comment (hmosvoll) [inline src/scripts/main/dialog/index.js:20]: Remove this comment if it is already fixed.

File: src/scripts/main/sheet/index.test.js [hunk 15]
```diff
@@ -376,9 +335,6 @@ describe("scripts: sheet", () => {
 
             sheet.close("demo-sheet");
             jest.runAllTimers();
-
-            expect(renderedSheet.classList).not.toContain("sheet-open");
-            expect(Object.keys(renderedToast.style._values)).not.toContain("margin-right");
```
Comment (hmosvoll) [inline src/scripts/main/sheet/index.test.js:381]: Should there be an expect here?

**Review by hmosvoll — COMMENTED** [review]
Nice.

_Review done in Slack huddle_

File: src/less/components/form.less (hunk 9 above)
Comment (PhungNg) [reply src/less/components/form.less:2]: Not all components require normalize.less

**Review by hmosvoll — APPROVED** [review]

**Pull request merged**